helm/resource_release_test.go

```diff
@@ -1758,6 +1758,45 @@ func TestAccResourceRelease_delete_regression(t *testing.T) {
 	})
 }
 
+func TestAccResourceRelease_destroy_ignoreReleaseNotFoundError(t *testing.T) {
```

Comment:
It's not really something we tend to test for let's remove the acceptance test and just go with the codechange.

Reply:
okay, I have removed it